src/Dialect/ONNX/ONNXOpsHelper.cpp

```diff
@@ -622,4 +625,60 @@ mlir::Type convertONNXTypeToMLIRType(
   llvm_unreachable("Unsupported data type encountered.");
 }
 
+// Convert an MLIR type to the correspoding ONNX type.
+int64_t mlirTypeToOnnxType(Type elemType) {
+  onnx::TensorProto::DataType onnxType = onnx::TensorProto::UNDEFINED;
```

Comment:
Reason to return int64_t instead of enum type onnx::TensorProto::DataType?

Reply:
the callers just need an int64_t number so I thought it was most efficient to do it like this and avoid that callers need to include onnx/onnx_pb.h

Reply:
Yes, it is good to avoid the include from onnx package. If the enum type is not explicit used, int64_t is fine.